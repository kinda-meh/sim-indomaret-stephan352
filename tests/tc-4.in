8 arrive
10 arrive
11 arrive
14 arrive
16 arrive
17 arrive
18 arrive
19 arrive
22 arrive
22 arrive
23 arrive
26 arrive
28 arrive
30 arrive
33 arrive
38 arrive
40 arrive
46 arrive
51 arrive
51 arrive
52 arrive
53 arrive
61 arrive
62 arrive
77 arrive
83 arrive
84 arrive
92 arrive
107 arrive
113 arrive
113 arrive
122 arrive
124 arrive
126 arrive
128 arrive
130 arrive
133 arrive
136 arrive
149 arrive
157 arrive
159 arrive
159 arrive
169 arrive
180 arrive
182 arrive
184 arrive
185 arrive
188 arrive
190 arrive
199 arrive
200 arrive
204 arrive
206 arrive
207 arrive
216 arrive
216 arrive
224 arrive
230 arrive
232 arrive
236 arrive
243 arrive
247 arrive
248 arrive
250 arrive
250 arrive
252 arrive
253 arrive
254 arrive
254 arrive
255 arrive
266 arrive
269 arrive
271 arrive
283 arrive
287 arrive
289 arrive
289 arrive
297 arrive
299 arrive
305 arrive
310 arrive
311 arrive
314 arrive
319 arrive
323 arrive
328 arrive
330 arrive
330 arrive
331 arrive
336 arrive
339 arrive
342 arrive
352 arrive
356 arrive
357 arrive
363 arrive
365 arrive
368 arrive
371 arrive
371 arrive
377 arrive
378 arrive
380 arrive
383 arrive
384 arrive
384 arrive
384 arrive
388 arrive
399 arrive
399 arrive
405 arrive
406 arrive
409 arrive
410 arrive
412 arrive
419 arrive
419 arrive
424 arrive
426 arrive
439 arrive
445 arrive
448 arrive
454 arrive
454 arrive
454 arrive
456 arrive
456 arrive
459 arrive
465 arrive
473 arrive
476 arrive
480 arrive
481 arrive
490 arrive
498 arrive
502 arrive
503 arrive
518 arrive
519 arrive
522 arrive
522 arrive
527 arrive
537 arrive
544 arrive
544 arrive
547 arrive
547 arrive
558 arrive
578 arrive
579 arrive
583 arrive
586 arrive
588 arrive
590 arrive
597 arrive
608 arrive
608 arrive
610 arrive
613 arrive
614 arrive
615 arrive
619 arrive
620 arrive
622 arrive
630 arrive
632 arrive
639 arrive
642 arrive
645 arrive
650 arrive
651 arrive
657 arrive
666 arrive
666 arrive
674 arrive
678 arrive
679 arrive
682 arrive
685 arrive
692 arrive
692 arrive
705 arrive
707 arrive
713 arrive
714 arrive
715 arrive
722 arrive
726 arrive
731 arrive
739 arrive
743 arrive
743 arrive
747 arrive
758 arrive
758 arrive
762 arrive
767 arrive
770 arrive
783 arrive
784 arrive
792 arrive
793 arrive
794 arrive
796 arrive
796 arrive
798 arrive
798 arrive
799 arrive
799 arrive
800 arrive
803 arrive
803 arrive
805 arrive
819 arrive
824 arrive
828 arrive
836 arrive
839 arrive
843 arrive
844 arrive
845 arrive
846 arrive
856 arrive
858 arrive
858 arrive
860 arrive
868 arrive
869 arrive
877 arrive
880 arrive
883 arrive
889 arrive
894 arrive
898 arrive
903 arrive
924 arrive
924 arrive
926 arrive
932 arrive
932 arrive
933 arrive
941 arrive
941 arrive
944 arrive
948 arrive
951 arrive
952 arrive
962 arrive
967 arrive
975 arrive
986 arrive
990 arrive
990 arrive
993 arrive
995 arrive
1000 arrive
