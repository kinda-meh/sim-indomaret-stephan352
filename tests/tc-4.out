0.491525 236 20
